src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -311,7 +324,11 @@ class Microsoft_Launcher : public PowertoyModuleIface
     void terminateProcess()
     {
         DWORD processID = GetProcessId(m_hProcess);
-        TerminateProcess(m_hProcess, 1);
+        if (TerminateProcess(m_hProcess, 1) == 0)
+        {
+            this->logger->error("Launcher process was not terminated. ErrorCode={}", GetLastError());
```

Comment:
btw there's a `get_last_error_message` in common.h for printing the error message itself

Reply:
done